src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -701,6 +701,9 @@ They can be configured through the settings file 'winget settings'.</value>
   <data name="UpdateNotApplicable" xml:space="preserve">
     <value>No applicable upgrade found.</value>
   </data>
+  <data name="UpdateNoPackagesFound" xml:space="preserve">
+    <value>No available upgrades found.</value>

```

Comment:
Since we are clarifying applicable vs available, should the message be updated to be more clear?

Reply:
Probably; @denelon - any suggestions?

Reply:
Do we use this message in aggregate with multiple packages anywhere or is this on a "one package only" flow?

If it's a one package only flow, what are all the things that could "cause" the error so we can capture the all?

```text
No applicable upgrade found. 
A newer package version is available in a configured source, but it does not apply to your system or requirements.
```

```text
No available upgrades found.
No newer package version is available in a configured source.
```

Reply:
> Do we use this message in aggregate with multiple packages anywhere or is this on a "one package only" flow?
> 
Depends on the scenario, but generally, both. If a user uses `winget upgrade --all` or `winget upgrade <id>`  there is a potential that no packages have newer versions available in configured sources. In tis case, I think the message `No newer package versions are available from the configured sources` is applicable to both, since it makes sense when talking about a single package or multiple packages.

If a user uses `winget upgrade --all`, the `No applicable upgrades` message is shown iff all packages which have a newer version available have no applicable installers.  If any package contains an applicable installer, the message is not shown and the inapplicable package is skipped silently.  If a user uses `winget upgrade <id>` then the message is shown when a newer package version is available but does not apply.  In this case I think it may be possible to check if the `--all` parameter is set and display a different message based on if it is a multiple-package or single-package flow? I like the `A newer package version is available in a configured source, but it does not apply to your system or requirements.` for single package, but for multiple it could be something like `Packages with newer versions are available from the configured sources, but do not apply to your system or requirements.` ?